plugins/woocommerce/client/admin/client/launch-your-store/settings/style.scss

```
@@ -99,10 +99,30 @@
 	.site-visibility-settings-slotfill-section {
 		max-width: 650px;
 
-		&:last-child {
+		// Separate consecutive sections rather than relying on the last child,
+		// so adding a new section (e.g. the site-visibility-badge toggle) does
+		// not remove the spacing between the sections above it.
+		& + .site-visibility-settings-slotfill-section {
```

Stylelint (17.13.0)

[error] Unnecessary nesting selector (&) (scss/selector-no-redundant-nesting-selector)

(scss/selector-no-redundant-nesting-selector)